panther_analysis_tool/rule.py

```diff
@@ -162,6 +162,12 @@ def __init__(self, config: Mapping):
                 values.sort()
             self.detection_reports = config["reports"]
 
+        if "suppressions" in config:
```

Comment:
Not a problem for sure, it seems that just a more compact version could also work here: 

```python
self.detection_suppressions = config.get("suppressions", [])
```